### PR TITLE
ADD: Adding new function to validate that a file or DataSet meet ARM …

### PR DESCRIPTION
**Pull request by AdamTheisen**
…standards

- [x] Closes #xxxx
- [x] Tests added
- [x] Documentation reflects changes
- [x] PEP8 Standards or use of linter
- [x] Xarray Dataset or DataArray variable naming follows 'ds' or 'da' naming

### CODE REVIEW

**Comment by AdamTheisen** [comment]
Going to add in one more test here

**Review by zssherman — APPROVED** [review]

File: act/utils/io_utils.py [hunk 1]
```diff
@@ -376,3 +380,190 @@ def generate_movie(images, write_filename=None, fps=10, **kwargs):
...
     return str(write_filename)
+
+
+def arm_standards_validator(file=None, dataset=None, verbose=True):
+    """
+    ARM Data Validator (ADV) - Checks to ensure that ARM standards are being followed
+    in the files or dataset passed to it.  Note, this includes a minimal set of
+    standards that it checks against
+
+    ...
```
Comment (zssherman) [inline act/utils/io_utils.py:391]: Can we remove this line and keep one space between the intro and parameter header

Comment (zssherman) [reply act/utils/io_utils.py]: Oh good catch haha! Thanks for doing that!

**Comment by zssherman** [comment]
Looks good to me @AdamTheisen ! Merging.

**Pull request merged**